blueprints/load-scheduling/elasticsearch/bundle.libsonnet

```diff
@@ -8,9 +8,11 @@ local config = blueprint.config;
 function(params) {
   local c = std.mergePatch(config, params),
 
+  local policyName = c.policy.policy_name,
+  local promqlQuery = 'avg(avg_over_time(elasticsearch_node_thread_pool_tasks_queued{policy_name="%(policy_name)s", infra_meter_name="elasticsearch", thread_pool_name="search"}[30s]))' % { policy_name: policyName },
```

Comment:
Can change the query to use a single aggregation `avg` which will get calculated based on latest datapoints for each series. No need for `avg_over_time` on 30sec interval